examples/config/ci/coordinator.json

```diff
@@ -88,7 +87,7 @@
   },
   "NumUsersInc": 8,
   "NumUsersDec": 8,
-  "RestTimeSec": 1,
```

Comment:
@agarciamontoro Not sure why this differed from our historical default of 2 seconds.

Reply:
Hmmm, I don't remember the reason either, to be honest. Maybe it was to try and make the coordinator converge sooner? I'm not really sure. About removing `UpdateIntervalMs` change: CI is still running an old version of the tool, but the default is 2000 in any case, so we don't expect any change there, right?

Reply:
Actually, chaning RestTimeSec from 2 to 1 will make things different, so I would move that change to another PR (after having checked the tests run normally)

Reply:
@agarciamontoro Sounds good, I can revert this bit. For my own understanding, is this file used as a source for our CI runs?

Reply:
Yes it is. We should probably move it out of `examples/`, if that's confusing.